servers/dsub/jobs/controllers/utils/generator.py

```diff
@@ -0,0 +1,164 @@
+import datetime
+
+from flask import current_app, request
+from dsub.commands import ddel, dstat
+from dateutil.tz import tzlocal
+
+from jobs.common import execute_redirect_stdout
+from jobs.controllers.utils import extensions, failures, job_ids, job_statuses, labels, providers, query_parameters
+from jobs.models.query_jobs_result import QueryJobsResult
+
+_MAX_PENDING_TIME_DAYS = 7
```

Comment:
At this point, it would probably make sense to name this more generally. e.g. MAX_RUNTIME_DAYS.

Reply:
Accepted.